src/Athena/Controllers/api/HomeController.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Athena.Controllers.api
+{
+    public class HomeController : Controller
```

Comment:
What is the purpose of this controller? Looks like a template MVC controller. Should probably be removed?